### PR TITLE
Enable loading multiple scripts in the repl

### PR DESCRIPTION
**Pull request by yotamN**
Instead of only loading one file, let the user pass the --load flag multiple times and concatenate them together.

### CODE REVIEW

File: frida_tools/repl.py [hunk 1]
```diff
@@ -197,13 +195,16 @@ def _stop(self):
             if not self._quiet:
                 self._print("\nThank you for using Frida!")
 
+        def _get_script_name(self, path):
+            return os.path.splitext(os.path.basename(path))[0]
```
Comment (oleavr) [inline frida_tools/repl.py:199]: This should go below the code that is using it. (Newspaper code structure.)

**Review by oleavr — CHANGES_REQUESTED** [review]
Yay, great feature. Thanks for doing this!

File: frida_tools/repl.py (hunk 1 above)
Comment (yotamN) [reply frida_tools/repl.py]: Fixed and rebased against main

**Comment by oleavr** [comment]
Thanks! Afraid we have some conflicts now ⚔️

**Comment by yotamN** [comment]
I rebased

**Pull request merged**